get_instagram.py

```diff
@@ -0,0 +1,24 @@
+from InstagramAPI import InstagramAPI
+from key import *
+
+
+def user(username):
+	api = InstagramAPI(username=user_name, password=password)
+	api.login()
+	try:
+		api.searchUsername(username)
```

Comment:
Why are you doing such complicated things?

Reply:
I do not know how to make it easier

Reply:
read sources careful

Reply:
done